Utils/PropertyWrapper/PropertyWrapperTests/Responses/PaymentResponse.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  PaymentResponse.swift
+//  PropertyWrapperTests
+//
+//  Created by Batuhan Saka on 22.09.2021.
+//  Copyright © 2021 Trendyol. All rights reserved.
+//
+
+import PropertyWrapper
+
+struct PaymentResponse: Decodable {
+    let success: Bool
+    @StringOrInt var orderId: String?
```

Comment:
other codable wrappers like `@DefaultCodable<DefaultFalse> var isAdmin: Bool`. Can you update `@StringOrInt` wrapper.